flare/examples/README.md

```diff
@@ -0,0 +1,21 @@
+# Flare Example
+
+This folder contains a small example of Flare's usage.
+
+## Training
+
+In `train.py`, a simple linear regression model is trained
+on the wine quality dataset. Before training, Flare's
+`baseline` method is invoked using the training dataframe.
+This creates our baseline files (`statistics.json` and
+`constraints.json`). Finally, the training script saves
+the trained model for later use.
+
+## Inference
+
+In `infer.py`, an example of using Flare at inference time
+is shown. First, the Slack alert target is configured, as
+shown the [quickstart](/flare#2-annotate-your-inference-code).
```

Comment:
I was just bitten by this. I suggest as an ongoing discipline that we place static link anchors into the markdown `<a name="static-name"></a>` and then use that after the # in links like this.

The issue is that if we add a step or change headings then this `2-...` will break